src/XMakeBuildEngine/Logging/BaseConsoleLogger.cs

```diff
@@ -1020,7 +1020,9 @@ public virtual void Initialize(IEventSource eventSource)
 
             if (showOnlyWarnings || showOnlyErrors)
             {
-                _showSummary = false;
+                if (!_showSummary.HasValue)
+                    _showSummary = false;
+
                 this.showPerfSummary = false;
```

Comment:
I didn't because performance summary wasn't a `bool?`. `summary` has the option of `summary`, `nosummary`, or not specified. The perf one doesn't so I left as is.